Gems/RecastNavigation/Code/Source/Misc/RecastHelpers.h

```diff
@@ -64,13 +64,12 @@ namespace RecastNavigation
         int m_tileX = 0; // tile coordinate within the navigation grid along X-axis
         int m_tileY = 0; // tile coordinate within the navigation grid along Y-axis
 
+        AZStd::function<void(AZStd::shared_ptr<TileGeometry>)> m_callback;
```

Comment:
question: Why make the name more generic? It is used for more than just tiles being loaded?

Reply:
No good reason. :) I changed a name and added a comment, too.